src/test/java/kitchenpos/bo/TableBoTest.java

```diff
@@ -0,0 +1,176 @@
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.BDDMockito.given;
+
+@ExtendWith(MockitoExtension.class)
+class TableBoTest {
+
+    @Mock
+    private OrderTableDao orderTableDao;
+
+    @Mock
+    private OrderDao orderDao;
+
+    @InjectMocks
+    private TableBo tableBo;
+
+    private OrderTable orderTable;
+    private List<String> orderStatusList;
+
+    @BeforeEach
+    void setUp() {
+        orderTable = new OrderTable();
+        orderTable.setId(1L);
+        orderTable.setNumberOfGuests(3);
+        orderTable.setEmpty(false);
+        orderStatusList = Arrays.asList(OrderStatus.COOKING.name(), OrderStatus.MEAL.name());
+    }
+
+    @DisplayName("테이블을 생성할 수 있다.")
+    @Test
+    void create() {
+        // given
+        given(orderTableDao.save(orderTable)).willReturn(orderTable);
+
+        // when
+        final OrderTable actual = tableBo.create(orderTable);
+
+        // then
+        assertThat(actual).isNotNull();
+        assertThat(actual.getNumberOfGuests()).isEqualTo(orderTable.getNumberOfGuests());
+    }
+
+    @DisplayName("테이블을 검색할 수 있다.")
+    @Test
+    void list() {
+        // given
+        given(orderTableDao.findAll())
+                .willReturn(Collections.singletonList(orderTable));
+
+        // when
+        List<OrderTable> orderTableList = tableBo.list();
+        OrderTable actual = orderTableList.get(0);
+
+        // then
+        assertThat(actual).isNotNull();
+        assertThat(actual.getNumberOfGuests()).isEqualTo(orderTable.getNumberOfGuests());
+    }
+
+    @DisplayName("테이블의 상태를 변경 할 수 있다")
+    @Test
+    void changeEmpty() {
+        // given
+        given(orderTableDao.findById(anyLong()))
+                .willReturn(Optional.of(orderTable));
+        given(orderDao.existsByOrderTableIdAndOrderStatusIn(orderTable.getId(), orderStatusList))
+                .willReturn(false);
+        given(orderTableDao.save(any(OrderTable.class)))
+                .willReturn(orderTable);
+
+        // when
+        OrderTable actual = tableBo.changeEmpty(orderTable.getId(), orderTable);
+
+        // then
+        assertThat(actual.isEmpty()).isEqualTo(orderTable.isEmpty());
+    }
+
+    @DisplayName("테이블을 빈테이블로 변경하려면 주문상태가 완료여야한다.")
+    @Test
+    void changeTableShouldCompleteStatus() {
+        // given
+        given(orderTableDao.findById(anyLong()))
+                .willReturn(Optional.of(orderTable));
+        given(orderDao.existsByOrderTableIdAndOrderStatusIn(orderTable.getId(), orderStatusList))
+                .willReturn(true);
+
+        // then
+        assertThatExceptionOfType(IllegalArgumentException.class)
+                .isThrownBy(()-> tableBo.changeEmpty(orderTable.getId(), orderTable));
+    }
+
+    @DisplayName("테이블을 빈테이블로 변경하려면 테이블이 존재해야한다.")
+    @Test
+    void changeTableShouldExist() {
+        given(orderTableDao.findById(anyLong()))
+                .willThrow(IllegalArgumentException.class);
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
                .willReturn(Optional.empty());
```